package.json

```diff
@@ -39,10 +42,14 @@
         "pino-pretty": "^10.0.0",
         "rome": "12.1.3",
         "ts-node": "^10.9.1",
+        "tsconfig-paths": "^4.2.0",
         "tsup": "^6.6.0",
         "typescript": "^5.0.4",
         "vite": "^4.4.2",
         "vitest": "^0.34.6"
     },
-    "packageManager": "^pnpm@8.6.0"
+    "packageManager": "^pnpm@8.6.0",
+    "resolutions": {
```

Comment:
Whats this for?

Reply:
When running the examples, I was receiving this error:
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/lgahdl/Documents/b-sdk/node_modules/.pnpm/get-port@6.1.2/node_modules/get-port/index.js from /Users/lgahdl/Documents/b-sdk/node_modules/.pnpm/@viem+anvil@0.0.6/node_modules/@viem/anvil/dist/cjs/pool/createPool.js not supported.
```
This error occurs because "@viem/anvil" imports other package "get-port" using cjs require, and get-port v6+ allows only ES import, if we had webpack or babel, this would be very easier to fix, but in my case I could only fix making the "get-port"version 5.1.1, I don't think this is the best solution, so I asked for Alberto's help, he may have a better approach for that.